ui/src/controller/track_decider.ts

```diff
@@ -406,6 +406,20 @@ class TrackDecider {
     }
   }
 
+  async groupCounterTracks(): Promise<void> {
+    for (let index = 0; index < this.tracksToAdd.length; index++) {
```

Comment:
nit: `track` could be used but isn't at lines 417 abd 420. More generally, index isn't used for anything but to iterate over the list. Might be clearner to use `forEach` or use `for...of`